docs/user-guide/query/gremlin/gremlin.md

```diff
@@ -250,7 +250,14 @@ a table of how different parts are mapped is as follows:
 | Vertex | Vertex with default label of `id` |
 | Entity | Vertex |
 | Edge | Edge |
-| Edge ID | A list with the source and destination of the Edge e.g. `[dest, source]` |
+
+In Gafferpop Edge ID's must be made up of a list containing either source and destination
+IDs, e.g. `[source, dest]`, or source, label and destination, e.g. `[source, label, dest]`.
+In a seeded query these should be formatted like so `g.E("[source, dest]")` or
+`g.E(["[source1, dest1]","[source2, label, dest2]"])`.
+
+Note that if using TSTV values for seeds or property values these must be in the
```

Comment:
Better to put this as otherwise expanding the initialism gives "Type Sub Type Value values".
```suggestion
Note that if using TypeSubTypeValue for seeds or property values these must be in the
```